Source/EasyNetQ/ConnectionConfiguration.cs

```diff
@@ -90,21 +92,29 @@ private void SetDefaultClientProperties(IDictionary<string, object> clientProper
             var hostname = Environment.MachineName;
             var product = Product ?? applicationName;
             var platform = Platform ?? hostname;
+            var name = Name ?? applicationName;
```

Comment:
is this wanted? we fallback on applicationName if the connection name is not present?